caffe2/utils/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ list(APPEND Caffe2_CPU_SRCS
 # ---[ threadpool/pthreadpool* is a local modification of the NNPACK
 # pthreadpool with a very similar interface. Neither NNPACK, nor this
 # thread pool supports Windows.
-if (NOT MSVC AND (NOT INTERN_BUILD_MOBILE OR BUILD_CAFFE2_MOBILE))
+if (NOT MSVC)
```

Comment:
does it mean that we'd also link with this shim on server-side? it should be fine, but are there any special effects from it for the regular NNPACK?

Reply:
probably already linking this shim for server build - note that we haven't changed how things work for server build (prior to this change it was bypassing mobile-build unless it's old caffe2-mobile build)